### DISCUSSION TITLE
MSCS%40NYU%20Tandon/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#271] [Announcements]
# MSCS%40NYU%20Tandon/

None

https://opencs.app/MSCS%40NYU%20Tandon/

**Comment by SAKUrie** [comment]
24Fall录取较为玄学，绩点过高或者过低都会被劝转CE。

**Comment by RowenaZnS** [comment]
25fall有算高gpa非科班选手也是cs不录被劝转CE, 最后被Courant CS录取
